RubeesEat/Pages/UserHomepage.cshtml

```diff
@@ -0,0 +1,17 @@
+@page
+@using System.Globalization
+@using System.Runtime.InteropServices.JavaScript
+@inject IBillRepository billRepository
+@inject IPersonRepository personRepository
+
+Hello, @personRepository.GetCurrentUser().FirstName!
+
+@{
+    var amount = billRepository.GetBalance(personRepository.GetCurrentUser().Id);
+    var isPositive = amount >= 0;
+    var displayColor = isPositive ?  "green" : "red";
+    var sign = isPositive ?  "+" : "-";
+    var amountAsString = sign + " " + amount.ToString("F2") + " €";
```

Comment:
If you use the `var amountAsString = sign + " " + amount.ToString("C");` format, it looks better + it automatically adds the currency

Reply:
Currency should always be Euro. C changes to a different Currency in a different country, depending on settings.